app/views/events/index.html.erb

```diff
@@ -0,0 +1,79 @@
+<p id="notice"><%= notice %></p>
+<div class="event-welcome">
+  <span>
+    Welcome: <b><%= session[:username] %></b> 
+  </span>
+   <a class="my-account" href= <%= "/users/#{session[:user_id]}" %> >My Account</a>
+</div>
+
+<div class="table-container">
+  <div class="table-1">
+    <h1 class="table-head">Upcoming Events</h1>
+
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Description</th>
+          <th>Date</th>
+          <th>Location</th>
+          <th>Creator</th>
+          <th colspan="3"></th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <% @future.each do |event| %>
+            <tr class="table-row">
+              <td><%= event.title %></td>
+              <td><%= event.body %></td>
+              <td><%= event.date %></td>
+              <td><%= event.location %></td>
+              <td><%= event.creator.name %></td>
+              <td ><span class="link"><%= link_to 'Show', event %></span></td>
+              <% if Attendance.where(user_id: session[:user_id], event_id: event.id).empty?%>
```

Comment:
That's not good. Why? Because of two reasons:
1. It's a logic in views
2. It's a model query in a views

- For logic, you must implement it into a view helpers
- For the model query, write a method into the model for that. In the User model will be preferable, so that you can call it like this
```rb
current_user.attend?(event_id)
```